test/JDBC/expected/BABEL-3292.out

```diff
@@ -0,0 +1,173 @@
+drop table if exists babel_3292_t1
+go
+
+drop table if exists babel_3292_t2
+go
+
+create table babel_3292_t1(a1 int PRIMARY KEY, b1 int, c1 int)
+go
+
+create index index_babel_3292_t1_b1 on babel_3292_t1(b1)
+go
+
+create index index_babel_3292_t1_c1 on babel_3292_t1(c1)
+go
+
+create table babel_3292_t2(a2 int PRIMARY KEY, b2 int)
+go
+
+create index index_babel_3292_t2_b2 on babel_3292_t2(b2)
+go
+
+set babelfish_showplan_all on
+go
+
+/*
+ * Run a SELECT query without any hints to ensure that un-hinted queries still work.
+ * This also ensures that when the SELECT query is not hinted it produces a different plan(bitmap heap scan and bitmap index scan)
+ * than the index scan that we're hinting in the query below. This verifies that the next test is actually valid.
+ * If the planner was going to choose a sequential scan anyway, the next test wouldn't actually prove that hints were working.
+ */
+select * from babel_3292_t1 where b1 = 1
+go
+~~START~~
+text
+Query Text: select * from babel_3292_t1 where b1 = 1
+Bitmap Heap Scan on babel_3292_t1  (cost=4.23..14.79 rows=10 width=12)
```

Comment:
Let's remove cost part from the expected result using
```
select set_config('babelfishpg_tsql.explain_costs', 'off', false);
select set_config('babelfishpg_tsql.explain_costs', 'on', false);
```

The reasons are
- costs are not our objectives
- costs can be changed by other factors such as vacuum analyze or PG versions

Reply:
Added the configuration so that costs are no longer part of the query plan explanation